app/editor/src/store/hooks/storage/useStorage.ts

```diff
@@ -14,7 +14,7 @@ interface IStorageController {
     onUploadProgress?: (progressEvent: any) => void,
   ) => Promise<IItemModel>;
   download: (locationId: number, path: string) => Promise<unknown>;
-  stream: (locationId: number, path: string) => Promise<unknown>;
+  stream: (locationId: number, path: string) => Promise<string>;
```

Comment:
Why is the stream returning a string?

Reply:
Updated, initially return a string as the endpoint return a base 64 string, thanks